### PR TITLE
add grad urls

### LINKED ISSUE #3298: Get graduation report links into Test Pilot

### PR DESCRIPTION
**Pull request by johngruen**
FIxes #3298

### CODE REVIEW

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3394/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/39f405e28247d217093bdc94dcd515b20710e88e/

**Comment by pdehaan** [comment]
All of the graduation report URLs resolved and didn't 404/fail. 
Noticed a couple TODOs buried in one of them, but they're all still drafts and can be finished whatever (as long as it's before the 13th, I s'pose).

**Comment by johngruen** [comment]
that's the plan :)

File: locales/en-US/experiments.ftl [hunk 1]
```diff
@@ -212,4 +212,4 @@ notesNewsupdatesNotesupdate1Content = We dramatically improved Notes by switchin
 sendNewsupdatesSendupdate1Title = Send with Safari and Edge!
 sendNewsupdatesSendupdate1Content = Send 1.1.0 just shipped with enhanced browser support and a few other improvements as well.
 sendNewsupdatesSendupdate2Title = Learning from Send
-sendNewsupdatesSendupdate2Content = We just conducted our first A/B test to learn more about Send’s user interface. Get the results.
+sendNewsupdatesSendupdate2Content = We just conducted our first A/B test to learn more about Send’s user interface. Get the results.
```
Comment (pdehaan) [inline locales/en-US/experiments.ftl:215]: This seems like an odd change. Not sure if something auto-trimmed the last blank line.

**Comment by pdehaan** [comment]
It currently looks like we have 5 active experiments, with 3 of them ending in a couple weeks. Do we need to make **Notes** non-featured? Not sure if it will look weird with one featured experiment (Notes), and then just one other experiment (**Send**) hanging out at the bottom in the "Try all of our experiments" section.

**Review by chuckharmston — APPROVED** [review]
Looks fine to me. Holding on @flodolo's approval.

**Comment by chuckharmston** [comment]
screen shot 2018-03-04 at 19 44 34

@johngruen you should check your system clock.

**Review by flodolo — APPROVED** [review]
Sorry, didn't think it was necessary, since it's only adding a blank line at the end (which is generally a good idea)

**Comment by chuckharmston** [comment]
@flodolo thanks!

We have you as a required-review [`codeowner`](https://help.github.com/articles/about-codeowners/) for any localization changes. If you'd prefer, I can mark yo uas not required?

**Comment by flodolo** [comment]
> We have you as a required-review codeowner for any localization changes. If you'd prefer, I can mark yo uas not required?

I prefer to keep that, since it pings me automatically for any change. But I don't think it's required, just flagged? i.e. I don't think it would be blocking a merge, if it is I'm happy to explicitly review all of them

**Pull request merged**